optuna_dashboard/ts/components/GraphHistory.tsx

```diff
@@ -69,6 +69,24 @@ export const GraphHistory: FC<{
       theme.palette.mode,
       markerSize
     )
+    const element = document.getElementById(plotDomId)
+    if (element != null && studies.length >= 1) {
+      // @ts-ignore
+      element.on("plotly_click", function (data) {
+        if (data.points[0].data.mode !== "lines") {
+          const link =
+            URL_PREFIX +
+            `/studies/${
+              studies[Math.floor(data.points[0].curveNumber / 2)].id
+            }/trials?numbers=${data.points[0].x}`
+          window.location.href = link
```

Comment:
Could you use `useNavigate()` to avoid the page refresh?

https://github.com/optuna/optuna-dashboard/blob/2f99e3b09c4adf280106d668e66f8164a0665e32/optuna_dashboard/ts/components/TrialList.tsx#L333

```suggestion
          navigate(
            URL_PREFIX +
            `/studies/${
              studies[Math.floor(data.points[0].curveNumber / 2)].id
            }/trials?numbers=${data.points[0].x}`
          )
```